### DISCUSSION TITLE
Crazyflie with Lighthouse deck does not take off after landing once and needs to be restarted.

### DISCUSSION THREAD

**Discussion by Reginald-Jeeves** [bitcraze/discussions#782] [Q&A]
I have attached the lighthouse deck and set up two base stations (v2) in my 2x2x2 flying area. The first time I run the script motion_flying.py, the cf takes off and lands as expected. As soon as I run the script again it does not take off. When I try to do it using the cfclient the flight control is grey and shows as disabled, upon hovering I get the message that the crazyflie cannot fly and needs to be reset. When I switch it off and back on it flies again. I am curious to know if this is normal and it needs a restart everytime it completes one takeoff and landing or is there something wrong with my setup/execution.

**Comment by krichardsson** [comment]
Hi, 
this is a problem we found in the 2023.06 release that is fixed in https://github.com/bitcraze/crazyflie-firmware/pull/1296.
The problem is that the tumble detection is a bit too sensitive and the landing the CF think it has crashed.

Either get the latest firmware from the master branch, build and flash, or go back to the previous release.
